tests/test_testing.py

```diff
@@ -1,70 +1,69 @@
 import json
-from typing import Any, Callable, Dict, Union
+from typing import Any, Callable, Dict, List, Union
 
 import pytest
-from hypothesis import HealthCheck, given, settings
-from hypothesis import strategies as st
 from pydantic import BaseModel
 
 from starlite import HttpMethod, RequestEncodingType, Starlite, State, get
 from starlite.datastructures import Cookie
 from starlite.enums import ParamType
 from starlite.testing import RequestFactory, TestClient, create_test_request
-from tests import Person, PersonFactory
+from tests import Pet, PetFactory
 
 _DEFAULT_REQUEST_FACTORY_URL = "http://test.org:3000/"
 
-person = PersonFactory.build()
-
-
-@settings(suppress_health_check=HealthCheck.all())
-@given(
-    http_method=st.sampled_from(HttpMethod),
-    scheme=st.text(),
-    server=st.text(),
-    port=st.integers(),
-    root_path=st.text(),
-    path=st.text(),
-    query=st.one_of(
-        st.none(),
-        st.dictionaries(keys=st.text(), values=st.one_of(st.lists(st.text()), st.text())),
-    ),
-    headers=st.one_of(st.none(), st.dictionaries(keys=st.text(), values=st.text())),
-    cookie=st.one_of(st.none(), st.text()),
-    content=st.one_of(
-        st.none(),
-        st.builds(Person),
-        st.dictionaries(keys=st.text(), values=st.builds(dict)),
-    ),
-    request_media_type=st.sampled_from(RequestEncodingType),
-)
-def test_create_test_request(
-    http_method: Any,
-    scheme: Any,
-    server: Any,
-    port: Any,
-    root_path: Any,
-    path: Any,
-    query: Any,
-    headers: Any,
-    cookie: Any,
-    content: Any,
-    request_media_type: Any,
-) -> None:
-    create_test_request(
+pet = PetFactory.build()
```

Comment:
I changed from `Person` to `Pet` because `Pet` is a simpler model so it's easier to test things like request body with content-type of `RequestEncodingType.URL_ENCODED`